src/Services/Application.php

```diff
@@ -63,4 +65,32 @@ protected function createApplication()
 
         return $app;
     }
+
+    /**
+     * Call artisan command and return code.
+     *
+     * @param string $command
+     * @param array  $parameters
+     *
+     * @return int
+     */
+    public function artisan($command, $parameters = [])
+    {
+        if (is_null($this->app)) {
+            return; // don't run when there is no application
+        }
+
+        return $this->code = $this->app['Illuminate\Contracts\Console\Kernel']->call($command, $parameters);
+    }
+
+    /**
+     * Set a user in laravel.
+     *
+     * @param UserContract $user
+     * @param null         $driver
+     */
+    public function be(UserContract $user, $driver = null)
+    {
+        $this->app['auth']->driver($driver)->setUser($user);
```

Comment:
If driver is Null, the default auth driver is used.  (this is set within your config/auth.php file)

This method is pulled strait from the default Laravel testing Application Trait.